tools/c7n_azure/c7n_azure/resources/front_door.py

```diff
@@ -32,3 +33,32 @@ class resource_type(ArmResourceManager.resource_type):
             'resourceGroup'
         )
         resource_type = 'Microsoft.Network/frontDoors'
+
+@FrontDoor.filter_registry.register('waf-not-enabled')
+class WebAppFirewallMissingFilter(Filter):
+    """Frontdoor check waf enabled on front door profiles for Classic_AzureFrontDoor
+
+    :example:
+
+    .. code-block:: yaml
+
+        policies:
+            name: test-frontdoor-waf-is-enabled
+            resource: azure.front-door
+            filters: [
+                - type: frontdoor-waf-is-enabled
+            ]
+
+    """
+    schema = type_schema('waf-not-enabled')
+
+    def process(self, resources, event=None):
+        client = self.manager.get_client()
+        results = []
+        for frontDoors in resources:
+            for frontendpoints in frontDoors['properties']['frontendEndpoints']:
+                frontendpoint = client.frontend_endpoints.get(
```

Comment:
that's three different variable styles in three lines, please use a consistent style ideally matching extant in the provider.